Hi,

Thank you very much for your efforts!

I have couple notes about your changes:
1. You forgot to import `os` module into `debug_toolbar_htmltidy/tests/tests.py` which caused import errors (use pyflakes)
2. Why you don't respect PEP8 formatting of entire code?
3. Tests failed with:

```
    test_apperance_builder (debug_toolbar_htmltidy.tests.tests.HTMLValidationDebugPanelTestCase) ... FAIL
    test_media (debug_toolbar_htmltidy.tests.tests.HTMLValidationDebugPanelTestCase) ... ok
    test_validator_counters (debug_toolbar_htmltidy.tests.tests.HTMLValidationDebugPanelTestCase) ... FAIL

    ======================================================================
    FAIL: test_apperance_builder (debug_toolbar_htmltidy.tests.tests.HTMLValidationDebugPanelTestCase)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/Users/gabonsky/Temp/django-dtpanel-htmltidy/debug_toolbar_htmltidy/tests/tests.py", line 109, in test_apperance_builder
        self.assertEqual(len(builded_errors), 41)
    AssertionError: 8 != 41

    ======================================================================
    FAIL: test_validator_counters (debug_toolbar_htmltidy.tests.tests.HTMLValidationDebugPanelTestCase)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/Users/gabonsky/Temp/django-dtpanel-htmltidy/debug_toolbar_htmltidy/tests/tests.py", line 96, in test_validator_counters
        self.assertEqual(panel.warns_count, 38)
    AssertionError: 6 != 38
```

It's easy to reproduce:
1. Clone your repo into new location
2. Create new virtual environment and activate it
3. Execute `python setup.py test`

If you have time, please fix it and update your pull request. I will be glad to merge it to mainstream.

--- a/debug_toolbar_htmltidy/tests/tests.py
+++ b/debug_toolbar_htmltidy/tests/tests.py
@@ -6,6 +6,8 @@ from django.test import TestCase
 
 from dingus import Dingus
 
+import os
+
 
 class Settings(object):
     """Allows you to define settings that are required
@@ -34,6 +36,11 @@ class BaseTestCase(TestCase):
     def setUp(self):
         settings.DEBUG = True
         settings.DEBUG_TOOLBAR_PANELS = self.panels_list
+        settings.TEMPLATE_DIRS = (
+                os.path.join(
+                    os.path.dirname(os.path.abspath(__file__)),
+                    'templates/'),
+                )
 
         request = Dingus('request')
         toolbar = DebugToolbar(request)

